src/main/java/org/pac4j/springframework/web/CallbackController.java

```diff
@@ -46,18 +40,16 @@ public class CallbackController {
     @RequestMapping("${pac4j.callback.path:/callback}")
     public Mono<Void> callback(final ServerWebExchange serverWebExchange) {
 
+        final SpringWebFluxFrameworkParameters frameworkParameters = new SpringWebFluxFrameworkParameters(serverWebExchange);
+
         final long t0 = System.currentTimeMillis();
         try {
 
-            final HttpActionAdapter bestAdapter = FindBest.httpActionAdapter(null, config, SpringWebfluxHttpActionAdapter.INSTANCE);
-            final CallbackLogic bestLogic = FindBest.callbackLogic(callbackLogic, config, DefaultCallbackLogic.INSTANCE);
-
-            final SpringWebfluxWebContext context = (SpringWebfluxWebContext) FindBest.webContextFactory(null, config, SpringWebfluxWebContextFactory.INSTANCE).newContext(serverWebExchange);
-            final SessionStore sessionStore = FindBest.sessionStoreFactory(null, config, SpringWebfluxSessionStoreFactory.INSTANCE).newSessionStore(serverWebExchange);
+            FrameworkAdapter.INSTANCE.applyDefaultSettingsIfUndefined(config);
 
-            bestLogic.perform(context, sessionStore, config, bestAdapter, this.defaultUrl, this.renewSession, this.defaultClient);
+            config.getCallbackLogic().perform(config, this.defaultUrl, this.renewSession, this.defaultClient, frameworkParameters);
 
-            return context.getResult();
+            return Mono.empty();
```

Comment:
This won't work.

Remove the `result` property from the `SpringWebfluxWebContext` and make the `SpringWebfluxHttpActionAdapter` return `response.writeWith(Mono.just(data))` or `Mono.empty();`

Reply:
3 times.